sql/core/src/test/resources/sql-tests/inputs/datetime-legacy.sql

```diff
@@ -1,2 +1,3 @@
 --SET spark.sql.legacy.timeParserPolicy=LEGACY
---IMPORT datetime.sql
+--IMPORT date.sql
+--IMPORT timestamp.sql
```

Comment:
We should remove this test file in the future. The config `spark.sql.legacy.timeParserPolicy` is only for parsing and `datetime-parsing-legacy.sql` should provide sufficient test coverage for parsing with the legacy mode.